packages/plugin-e2e/src/models/pages/VariablePage.ts

```diff
@@ -22,11 +23,11 @@ export class VariablePage extends GrafanaPage {
   /**
    * Clicks the add new variable button and returns the variable edit page
    */
-  async clickAddNew() {
+  async clickAddNew(first?: boolean) {
```

Comment:
I guess the `first` arg is needed in case the dashboard has been saved during test execution? 

Anyway, doesn't feel right that the consumer need to pass this variable - plugin-e2e should be able to figure out what button to click. 

Not very pretty, but think this should work and replace the need for this arg.

```ts
    try {
      await this.getByGrafanaSelector(addVariableCTAV2(addVariableCTAV2Item)).click({ timeout: 500 });
    } catch (_) {
      await this.getByGrafanaSelector(newButton).click();
    }
```

Reply:
Great catch, I've updated this accordingly